sources/packages/forms/Dockerfile

```diff
@@ -1,54 +1,36 @@
-# Used by docker-compose.yml to deploy the formio application
-# (When modified, you must include `--build` )
-# -----------------------------------------------------------
+# Use Node image, maintained by Docker: hub.docker.com/r/_/node.
+FROM artifacts.developer.gov.bc.ca/docker-remote/node:20-alpine3.20
```

Comment:
Not a part of the PR. But was there a reason why alpine version was used here? and why not he redhat node-20.

Reply:
Alpine was already the one being used.
Our current docker file is based on the form.io one: https://github.com/formio/formio/blob/v4.2.4/Dockerfile